### ISSUE TITLE
does not support files above 4GB

### ISSUE THREAD

**Issue by bmwiedemann** [cyrus-and/httpfs#5]
The size in httpfs_getattr
and the offset in httpfs_read, httpfs_write, httpfs_truncate are only 32bit integers,
so can not work with files larger than 4GB (e.g. videos and VM images).
One could either use a 64bit int or 2x32bit or an ascii integer to represent large enough sizes/offsets.

**Comment by cyrus-and** [comment]
You're right, this problem too will possibly be solved in the full rewrite.

Apart from the function interface, I need a reliable way to implement this in PHP, AFAIK there are some problems for files bigger than 2/4GB. I should investigate that.

Thanks for reporting.
